modules/recieve_statustxt.py

```diff
@@ -0,0 +1,63 @@
+import time
+from pymavlink import mavutil
+from modules.common.data_encoding.message_encoding_decoding import decode_bytes_to_position_global
+from modules.common.data_encoding.metadata_encoding_decoding import decode_metadata
+from modules.common.data_encoding.worker_enum import WorkerEnum
+from modules.common.kml.kml_conversion import positions_to_kml
+from pathlib import Path
+
+CONNECTION_ADDRESS = "tcp:localhost:14550"
+DELAY = 1
+vehicle = mavutil.mavlink_connection(CONNECTION_ADDRESS, source_system=255, source_component=0)
+vehicle.wait_heartbeat()
+print("connected")
+
+positions = []
```

Comment:
Better to declare this inside the infinite loop, so you don't have to do it again later below.

Reply:
Done :)